src/plugins/codeStyle/index.ts

```diff
@@ -56,6 +56,11 @@ export default class CodeStyle {
                         );
                     }
                 }
+            },
+            PrintStatement: s => {
+                if (noPrint) {
```

Comment:
If not `off`

Reply:
So I made no-print rules a `RulesSeverity` and I actually meant to remove this if. 
`        'no-print'?: RuleSeverity;`
If you'd rather it be a different type lmk